src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -339,7 +339,7 @@ public virtual int Compare(string string1, string string2, CompareOptions option
             if (_invariantMode)
             {
                 if ((options & CompareOptions.IgnoreCase) != 0)
-                    return CompareOrdinalIgnoreCase(string1, 0, string1.Length, string2, 0, string2.Length);
+                    return CompareOrdinalIgnoreCase(string1, string2);
```

Comment:
I made (and reverted) a similar change here: https://github.com/dotnet/coreclr/pull/16223#discussion_r166832040 because I observed perf regression. Do you know if this change has any affects on perf?